### ISSUE TITLE
Template Editor: Mobile website (Iphone) allows you to leave your presentation without saving it.

### ISSUE THREAD

**Issue by RowanDarlison** [Rise-Vision/rise-vision-apps#968]
**How it currently works:** 
Mobile website (Iphone) allows you to leave your presentation without saving it.

**How it should work**
Should ask you to save your work before exiting.
https://app.zeplin.io/project/5c58a8a2b74a443081a59e09/screen/5c910af4c4045605c458e967

**Comment by ezequielc** [comment]
@sheadarlison  @RowanDarlison I found two instances of this issue:

1. When clicking on any navigation link from the Sandwich/Sidebar menu.
2. When doing a browser page reload.

In both cases we were not prompting users to save before leaving.

For the first one, which I think represents most of the cases, I managed to release a fix.
The second case is actually a Mobile Safari limitation, it doesn't allow us to show a confirmation modal on page reloads, the API for that is disabled. It works fine on other systems. I researched but could not find a workaround for it. 

Given the above, I am closing the issue but let me know if you think otherwise. Thanks!
